internal/common/rules/basicruleevaluator.go

```diff
@@ -0,0 +1,170 @@
+package rules
+
+import (
+	"fmt"
+	"github.com/pkg/errors"
+	"gonum.org/v1/gonum/mat"
+)
+
+func createVarList(variables []string) ([]float64, error) {
+	var variableVect []float64
+
+	for _, v := range variables {
+		if val, varOk := VariableMap[v]; varOk {
+			variableVect = append(variableVect, val.Values...)
+		} else {
+			return nil, errors.New(fmt.Sprintf("Variable: '%v' not found in global variable cache", v))
```

Comment:
Use errors.Errorf instead from GitHub.com/pkg/errors

Reply:
Agreed and changed